### ISSUE TITLE
Can‘t install

### ISSUE THREAD

**Issue by ASAPRockyy** [AnemoneTeam/Anemone#298]
I updated my ios 10.1.1 jailbreak to 11.3.1, I had some problems after jailbreaking, I deleted Electra with delectra and after that I manually installed some important tweaks (Substare and everything like that)
Every other tweak works but Anemone does not. What should I do? (I could send you a screenshot of the Cydia Process)

**Comment by gyz02** [comment]
Yes im having the same issue but i was on ios 9.3.3 and updated to 11.3.1 , i keep getting subprocess error ... (1)

**Comment by r1979-sys** [comment]
I need your update on selio to install Anemone 3
